homework-assignments-1/task-4.ts

```diff
@@ -0,0 +1,154 @@
+const ShippingCost: Record<string, number> = {
+    Tashkent: 10.8,
+    London: 15.7,
+    Toronto: 11.3,
+    Sydney: 14.7
```

Comment:
Not seeing New York here. Seems no sympathy for USA (joke). No need to pay attention to this